packages/entities/entities-shared/src/components/entity-empty-state/EntityEmptyState.vue

```diff
@@ -220,10 +245,14 @@ $entity-empty-state-max-width: calc(2 * #{$entity-empty-state-feature-card-width
     max-width: 640px; // limit width so the description stays readable if it is too long
 
     p {
-      margin: $kui-space-30;
+      margin: $kui-space-0;
```

Comment:
Can you confirm this doesn't mess up spacing when a `pricing` string is provided?

Reply:
with and without pricing 
<img width="934" alt="Screenshot 2025-01-31 at 9 23 18 PM" src="https://github.com/user-attachments/assets/59c40d82-073a-4263-8e76-297f820c32e6" />
<img width="1187" alt="Screenshot 2025-01-31 at 9 22 44 PM" src="https://github.com/user-attachments/assets/8ef6d36f-e25f-45b4-b398-4de4f3d8bcde" />